### ISSUE TITLE
Convert workbench editor actions to commands

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#59346]
The following editors are using old-school actions and should change to a model where a command is contributed to the editor title area:
* [x] [keybindings editor](https://github.com/Microsoft/vscode/blob/515c90a9fe1d8a28c0794420c3d82f7060bce73c/src/vs/workbench/parts/preferences/browser/keybindingsEditor.ts#L153) @sandy081 
* [x] [extension runtime editor](https://github.com/Microsoft/vscode/blob/515c90a9fe1d8a28c0794420c3d82f7060bce73c/src/vs/workbench/parts/extensions/electron-browser/runtimeExtensionsEditor.ts#L413) @isidorn 
* [x] [diff editor](https://github.com/Microsoft/vscode/blob/515c90a9fe1d8a28c0794420c3d82f7060bce73c/src/vs/workbench/browser/parts/editor/textDiffEditor.ts#L295) @bpasero

Example of a good contribution: [link](https://github.com/Microsoft/vscode/blob/515c90a9fe1d8a28c0794420c3d82f7060bce73c/src/vs/workbench/browser/parts/editor/editor.contribution.ts#L471)

Once this is in, the editor part can remove calls to `getActions()`, `getSecondaryActions()` from `Composite`.

**Comment by isidorn** [comment]
Assigning to october so I tackle my part then

**Comment by bpasero** [comment]
I pushed the changes for the diff editor to fix https://github.com/Microsoft/vscode/issues/59338

**Comment by isidorn** [comment]
I looked into the Running Extensions and that whole view is an old school pain.
I think this should be tackled via https://github.com/Microsoft/vscode/issues/57969 @ramya-rao-a 

I actually started to convert but requires some work since the actions currently in the title area of the running extensions view dynamically update their label and icons. So we would need to split them up in multiple commands.
Bottom line I think it does not deserve the effort

**Comment by bpasero** [comment]
@isidorn can you give an overview of how the actions change exactly? would it help if some actions moved into the "..." menu instead of being primary?

**Comment by isidorn** [comment]
Extension host profile actions changes from start debugging to stop debugging. [Code pointer](https://github.com/Microsoft/vscode/blob/0c4fa8a7b5f8ef2b59663b20d79df7776a697609/src/vs/workbench/parts/extensions/electron-browser/runtimeExtensionsEditor.ts#L535)
@bpasero if you are passionate about getting rid of this I can do it, but I just did not find it smart to invest in something we should actually get rid of and put in the extensions viewlet.
Moving into the `...` would not change much as the title needs to change still and that is not supported by commands, thus two seperate commands would be needed.

**Comment by bpasero** [comment]
@isidorn so it is just this one action? Then just add a context key for this scenario.

**Comment by sandy081** [comment]
Done

**Comment by bpasero** [comment]
Thanks!
